roles/install/vars/OS-RedHat.yml

```diff
@@ -0,0 +1,35 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+_community_sops_install_system_has_system: false
+_community_sops_install_system_has_system_latest_only: true
+_community_sops_install_system_has_github: true
+
+_community_sops_install_allow_downgrade: true
+
+_community_sops_install_system_packages:
+  - gnupg2
+  - age
```

Comment:
`age` is only available on Fedora, so it shouldn't be listed here. You should remove age from here and then get rid of OS-RedHat-{7,8}.yml.

Reply:
Ah, my hope was that it will be available in RHEL 9 maybe (unfortunately that's only available in CI with AZP right now). I was writing that part while being offline, so I couldn't check, and then later forgot... :)

Reply:
Fixed in a3e9f9b6ad2700b112a34b171a54d2c4e2f351ce.

Reply:
> Ah, my hope was that it will be available in RHEL 9 maybe (unfortunately that's only available in CI with AZP right now).

It could get added to EPEL 9, but I doubt that'll happen anytime soon.